src/internal/coredb/projects_test.go

```diff
@@ -122,25 +122,6 @@ func TestListProject(t *testing.T) {
 	}))
 }
 
-func TestDeleteAllProjects(t *testing.T) {
```

Comment:
Why delete this test?

Reply:
Delete All relies on `truncate` which isn't allowed when you have a foreign key dependency. We could write a delete all that just does a list and calls delete, but we don't currently have a use case for either of those functions. I figured in the meantime it was better to remove them.